docroot/modules/humsci/hs_migrate/src/EventSubscriber/EventsSubscriber.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
```

Comment:
Perhaps a file comment about what this subscriber is doing would be helpful here.

Reply:
I'll update the class comment since `@file` are not allowed in namespaced files https://www.drupal.org/docs/develop/standards/api-documentation-and-comment-standards#file